packages/core/src/event_delegation_utils.ts

```diff
@@ -34,16 +35,21 @@ export function invokeRegisteredListeners(event: Event) {
   }
 }
 
-export function setJSActionAttribute(nativeElement: Element, eventTypes: string[]) {
+export function setJSActionAttributes(nativeElement: Element, eventTypes: string[]) {
   if (!eventTypes.length) {
     return;
   }
   const parts = eventTypes.reduce((prev, curr) => prev + curr + ':;', '');
   const existingAttr = nativeElement.getAttribute(Attributes.JSACTION);
-  //  This is required to be a module accessor to appease security tests on setAttribute.
   nativeElement.setAttribute(Attributes.JSACTION, `${existingAttr ?? ''}${parts}`);
```

Comment:
nit: we can reuse the logic here (we can also drop the `const existingAttr = ...` line above):
```suggestion
  setJSActionAttribute(nativeElement, `${parts}`);
```

Reply:
(this may not be exactly correct, since there are `:;` symbols that we need to handle, this is more to illustrate the idea)

Reply:
i couldn't figure out how to share logic without it being awkward (due to the :; symbol). I don't know if it saves that many lines.